enterprise/server/hit_tracker_client/hit_tracker_client.go

```diff
@@ -0,0 +1,56 @@
+package hit_tracker_client
```

Comment:
might be worth adding a TODO to add the RPC impl (took me a sec to figure out that this was the intent, haha)

Reply:
sure thing